strawberry/types/info.py

```diff
@@ -15,7 +17,8 @@
 @dataclasses.dataclass
 class Info(Generic[ContextType, RootValueType]):
     field_name: str
-    field_nodes: List[FieldNode]
+    field_nodes: List[FieldNode]  # deprecated
+    field: SelectionField
```

Comment:
what do you think of renaming this to `current_field` or `selected_field`? 

not sure just `field` is a good name :) what do you think?

Reply:
Oh, I found a problem. It is possible to have multiple field nodes: if a client splits an identical field. So this will have to be a list, just like the original `field_nodes`.

As for naming, for the other nodes I basically just stripped the suffix `Node`, but that just leaves us with `Field` again. I guess I don't have a preference.

Reply:
`selected_fields` might be a good name then. Do we get multiple selection when doing a query like this:

```
{
  field {
    name
    name
  }
}
```

?

Reply:
Yes, exactly.

Renamed to `selected_fields: List[SelectedField]`.

Reply:
I'd have expected it to merge them :D interesting!

what about field aliases?

```
{
  field {
    name
    some: name
  }
}
```

Reply:
Aliases are resolved separately.

But apparently duplicate fields are allowed as long as they're identical (w.r.t. inputs), and then resolved once with multiple `field_nodes`. I don't know why anyone would need that feature, but it's there.